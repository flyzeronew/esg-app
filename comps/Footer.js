function Footer(props) {
    return (
            <footer className={props.act}>
                © TVBS Media Inc. All Rights Reserved <br/>聯利媒體股份有限公司
            </footer>
        )
}
export default Footer;