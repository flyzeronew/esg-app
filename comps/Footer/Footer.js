import styles from './Footer.module.css';

function Footer(props) {
  return (
    <footer className={styles.footer}>
      © TVBS Media Inc. All Rights Reserved <br />
      聯利媒體股份有限公司
    </footer>
  );
}
export default Footer;
