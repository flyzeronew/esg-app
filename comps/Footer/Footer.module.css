.footer{
    color: #636363;
    text-align: center;
    font-size: 14px;
    font-style: normal;
    font-weight: 400;
    line-height: normal;
    padding:10px 15px 32px;
    background-color: #fff;
}
.footer.color1{
    background-color: #F2F2F2;
}
.footer br{
    display: none;
}
@media (max-width:1023px) {
    .footer br{
        display: block;
    }
}