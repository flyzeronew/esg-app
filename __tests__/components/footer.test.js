import { render } from '@testing-library/react';
import Footer from './../../comps/Footer/Footer';

describe('Footer Component', () => {

  it("renders footer unchanged", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
  it('renders footer component with correct class', () => {
    const { getByText } = render(<Footer />);
    const footerElement = getByText(
      '© TVBS Media Inc. All Rights Reserved 聯利媒體股份有限公司'
    );
    expect(footerElement).toBeInTheDocument();
    expect(footerElement).toHaveClass('footer');
  });
});
