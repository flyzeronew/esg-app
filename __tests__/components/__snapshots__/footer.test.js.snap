// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component renders footer unchanged 1`] = `
<div>
  <footer
    class="footer"
  >
    © TVBS Media Inc. All Rights Reserved 
    <br />
    聯利媒體股份有限公司
  </footer>
</div>
`;
